index.bs

```diff
@@ -4639,6 +4639,121 @@ settings:</p>
  Objects</a> not part of a region</li>
 </ul>
 
+<p>The 'color' property on <a>WebVTT Internal Node Objects</a> that have one or more <a lt="WebVTT
+Node Object's applicable classes">applicable classes</a> matching those in the table below, must be
+set, as <a spec=html>presentational hints</a>, to the value in the second cell of the row of the
+table below whose first cell is the value of the corresponding <a lt="WebVTT Node Object's
+applicable classes">applicable class</a>:</p>
+
+<table class="complex data">
+ <thead>
+  <tr>
+   <th><a lt="WebVTT Node Object's applicable classes">applicable class</a></th>
+   <th>'color' value</th>
+  </tr>
+ </thead>
+ <tbody>
+  <tr>
+   <td><code>white</code></td>
+   <td>''rgba(255,255,255,1)''</td>
+  </tr>
+  <tr>
+   <td><code>green</code></td>
+   <td>''rgba(0,255,0,1)''</td>
+  </tr>
+  <tr>
+   <td><code>cyan</code></td>
+   <td>''rgba(0,255,255,1)''</td>
+  </tr>
+  <tr>
+   <td><code>red</code></td>
+   <td>''rgba(255,0,0,1)''</td>
+  </tr>
+  <tr>
+   <td><code>yellow</code></td>
+   <td>''rgba(255,255,0,1)''</td>
+  </tr>
+  <tr>
+   <td><code>magenta</code></td>
+   <td>''rgba(255,0,255,1)''</td>
+  </tr>
+  <tr>
+   <td><code>blue</code></td>
+   <td>''rgba(0,0,255,1)''</td>
+  </tr>
+  <tr>
+   <td><code>black</code></td>
+   <td>''rgba(0,0,0,1)''</td>
+  </tr>
+ </tbody>
+</table>
+
+<p class=note>The color for the class <code>green</code> is <em>not</em> the same as the CSS named
+color ''green'', but rather matches ''lime''. This is because this color has traditionally been called ''green'' in captioning and subitling.</p>
+
+<p class=note>Do not use the classes <code>blue</code> and <code>black</code> on the default dark background, since they result in unreadable text.</p>
```

Comment:
Maybe refer to WCAG guidance on color contrast as setting the requirements here, and in the context of captions on video, take account of the video's color too. There are plenty of other video/background/foreground color combinations that are not readable.

Reply:
Good idea thanks.